receiver/redisreceiver/documentation.md

```diff
@@ -55,7 +55,7 @@ metrics:
 
 | Name | Description | Type |
 | ---- | ----------- | ---- |
-| redis.version | Redis server's version. | Str |
+| redis.version | Redis server's version. | String |
```

Comment:
This shouldn't be changed

Reply:
This is auto generated.

Reply:
You need to rebase then

Reply:
I sycned my repo to main and did the rebase. But looks like this has been changed from Str to String, so the generated doc is still this way.

Reply:
It looks like I need to re-install mdatagen to re-generate the files. Mark this done